src/status.rs

```diff
@@ -1,1601 +1,331 @@
-    Code541 = 541,
-    /// 542 (unregistered)
-    Code542 = 542,
-    /// 543 (unregistered)
-    Code543 = 543,
-    /// 544 (unregistered)
-    Code544 = 544,
-    /// 545 (unregistered)
-    Code545 = 545,
-    /// 546 (unregistered)
-    Code546 = 546,
-    /// 547 (unregistered)
-    Code547 = 547,
-    /// 548 (unregistered)
-    Code548 = 548,
-    /// 549 (unregistered)
-    Code549 = 549,
-    /// 550 (unregistered)
-    Code550 = 550,
-    /// 551 (unregistered)
-    Code551 = 551,
-    /// 552 (unregistered)
-    Code552 = 552,
-    /// 553 (unregistered)
-    Code553 = 553,
-    /// 554 (unregistered)
-    Code554 = 554,
-    /// 555 (unregistered)
-    Code555 = 555,
-    /// 556 (unregistered)
-    Code556 = 556,
-    /// 557 (unregistered)
-    Code557 = 557,
-    /// 558 (unregistered)
-    Code558 = 558,
-    /// 559 (unregistered)
-    Code559 = 559,
-    /// 560 (unregistered)
-    Code560 = 560,
-    /// 561 (unregistered)
-    Code561 = 561,
-    /// 562 (unregistered)
-    Code562 = 562,
-    /// 563 (unregistered)
-    Code563 = 563,
-    /// 564 (unregistered)
-    Code564 = 564,
-    /// 565 (unregistered)
-    Code565 = 565,
-    /// 566 (unregistered)
-    Code566 = 566,
-    /// 567 (unregistered)
-    Code567 = 567,
-    /// 568 (unregistered)
-    Code568 = 568,
-    /// 569 (unregistered)
-    Code569 = 569,
-    /// 570 (unregistered)
-    Code570 = 570,
-    /// 571 (unregistered)
-    Code571 = 571,
-    /// 572 (unregistered)
-    Code572 = 572,
-    /// 573 (unregistered)
-    Code573 = 573,
-    /// 574 (unregistered)
-    Code574 = 574,
-    /// 575 (unregistered)
-    Code575 = 575,
-    /// 576 (unregistered)
-    Code576 = 576,
-    /// 577 (unregistered)
-    Code577 = 577,
-    /// 578 (unregistered)
-    Code578 = 578,
-    /// 579 (unregistered)
-    Code579 = 579,
-    /// 580 (unregistered)
-    Code580 = 580,
-    /// 581 (unregistered)
-    Code581 = 581,
-    /// 582 (unregistered)
-    Code582 = 582,
-    /// 583 (unregistered)
-    Code583 = 583,
-    /// 584 (unregistered)
-    Code584 = 584,
-    /// 585 (unregistered)
-    Code585 = 585,
-    /// 586 (unregistered)
-    Code586 = 586,
-    /// 587 (unregistered)
-    Code587 = 587,
-    /// 588 (unregistered)
-    Code588 = 588,
-    /// 589 (unregistered)
-    Code589 = 589,
-    /// 590 (unregistered)
-    Code590 = 590,
-    /// 591 (unregistered)
-    Code591 = 591,
-    /// 592 (unregistered)
-    Code592 = 592,
-    /// 593 (unregistered)
-    Code593 = 593,
-    /// 594 (unregistered)
-    Code594 = 594,
-    /// 595 (unregistered)
-    Code595 = 595,
-    /// 596 (unregistered)
-    Code596 = 596,
-    /// 597 (unregistered)
-    Code597 = 597,
-    /// 598 (unregistered)
-    Code598 = 598,
-    /// 599 (unregistered)
-    Code599 = 599,
+    /// [[RFC6585](https://tools.ietf.org/html/rfc6585)]
+    NetworkAuthenticationRequired,
+
+    /// A method not in the IANA HTTP status code registry.
+    UnregisteredStatusCode(u16),
```

Comment:
Since it's already got the StatusCode namespace, I'd leave this as just 'Unregistered'.